client/spa-pages/components/MapPage.tsx

```diff
@@ -378,22 +372,13 @@ const MapInner = ({ setPage }: Props) => {
 					handleChangedLocation={handleChangedLocation}
 				/>
 
-				{facCardIsOpen &&
-					(isMobile ? (
-						<FacilityCard
-							facCardDetails={facCardDetails}
-							facCardDistance={facCardDistance}
-							width={"86%"}
-							left={"7%"}
-						/>
-					) : (
-						<FacilityCard
-							facCardDetails={facCardDetails}
-							facCardDistance={facCardDistance}
-							width={"50%"}
-							left={"25%"}
-						/>
-					))}
+				{facCardIsOpen && (
+					<FacilityCard
+						items={items}
+						facCardDetails={facCardDetails}
+						facCardDistance={facCardDistance}
+					/>
+				)}

```

Comment:
Nice! It's good that we remove the device check here.

We can probably remove the `isMobile` assignment as well.